communication/youtube/README.md

```diff
@@ -0,0 +1,25 @@
+# YouTube Management Project (The Stream Team)
+
+The Kubernetes community is an open community. As such, numerous meetings are
+recorded and shared through the
+[Kubernetes YouTube](https://www.youtube.com/channel/UCZ2bu0qutTOM0tHYa_jkIwg)
+channel. Additionally, requests to live stream meetings are increasing as the
+community grows.
+
+## YouTube Channel Guidelines
+
+Please refer to [YouTube Channel Guidelines] with questions regarding YouTube.
```

Comment:
Missing the reference link at the bottom. 
```
[youtube channel guidelines]: ./youtube-guidelines.md
```

Having that at the bottom of the doc will fix it for both references to `[YouTube Channel Guidelines]` 👍